eventing/samples/github-source/README.md

```diff
@@ -0,0 +1,160 @@
+# GitHub Source example
+
+GitHub Source example shows how to wire GitHub events for consumption
+by a Knative Service.
+
+## Deployment Steps
+
+### Prerequisites
+
+You will need:
+
+1. An internet-accessible Kubernetes cluster with Knative Serving
+  installed. Follow the [installation
+  instructions](https://github.com/knative/docs/blob/master/install/README.md)
+  if you need to create one.
+  - Ensure Knative Serving is [configured with a domain
+    name](https://github.com/knative/docs/blob/master/serving/using-a-custom-domain.md)
+    that allows GitHub to call into the cluster.
+  - If you're using GKE, you'll also want to [assign a static IP address](https://github.com/knative/docs/blob/master/serving/gke-assigning-static-ip-address.md).
+1. Install [Knative
+  Eventing](https://github.com/knative/docs/tree/master/eventing).
+1. Create a `Channel`. You can use your own `Channel` or use the
+   provided sample, which creates a channel called `githubchannel`. If
+   you use your own `Channel` with a different name, then you will
+   need to alter other commands later.
+
+```shell
+kubectl -n default apply -f eventing/samples/github-source/channel.yaml
```

Comment:
Docs WG has a preference for spelling out the kubernetes flag names, e.g. `--namespace` and `--filename`. (Yes, this got missed on the other docs commits.)